src/shared/scrapers/AUS/SA/index.js

```diff
@@ -2,6 +2,20 @@ import * as parse from '../../../lib/parse.js';
 import * as fetch from '../../../lib/fetch/index.js';
 import maintainers from '../../../lib/maintainers.js';
 
+const getKey = rowLabel => {
+  const lowerLabel = rowLabel.toLowerCase();
+  if (lowerLabel.includes('confirmed cases')) {
+    return 'cases';
+  }
+  if (lowerLabel.includes('icu')) {
+    return 'hospitalized';
```

Comment:
random guess, if we have a standard name for this please let me know.

Reply:
We don't, but that's a good start!